packages/date-picker/test/keyboard-input.test.js

```diff
@@ -257,70 +264,39 @@ describe('keyboard', () => {
     });
 
     describe('focus date not in the viewport', () => {
-      let spy;
-
-      afterEach(() => {
-        spy.restore();
-      });
-
-      it('should focus date scrolled out of the view on input Tab', async () => {
-        // Move focus to the calendar
-        await sendKeys({ press: 'Tab' });
-        await nextRender(datepicker);
-
+      beforeEach(async () => {
         // Scroll to date outside viewport
         const date = new Date();
         date.setFullYear(date.getFullYear() - 1);
-        overlayContent.revealDate(date);
-        await waitForScrollToFinish(overlayContent);
+        overlayContent.revealDate(date, false);
+        await idleCallback();
```

Comment:
There seems to be an issue when using `sendKeys` does not move focus correctly:

1. When you put a `debugger` in the beginning of a test and immediately resume, it passes
2. When you run the test without `debugger`, it fails as the date does not get focused

Waiting for `idleCallback` seems to fix this when debugging these tests in Chrome.